### PR TITLE
Feature/weos 1124

### PR DESCRIPTION
**Pull request by RandyDeo**
* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Review by Aidan-E — DISMISSED** [review]
Blog created successfully with all parameters fulfilled.

File: features/create-content.feature [hunk 1]
```diff
@@ -1,4 +1,4 @@
-
+@skipped
```
Comment (akeemphilbert) [inline features/create-content.feature]: This should not be skipped I believe

File: end2end_test.go [hunk 2]
```diff
@@ -210,10 +210,15 @@ func aModelShouldBeAddedToTheProjection(arg1 string, details *godog.Table) error
 				}
 				if !strings.EqualFold(column.DatabaseTypeName(), cell.Value) {
 					return fmt.Errorf("expected to get type '%s' got '%s'", cell.Value, column.DatabaseTypeName())
-
 				}
 			//ignore this for now.  gorm does not set to nullable, rather defaulting to the null value of that interface
-			case "Null", "Default":
+			case "Null":
```
Comment (akeemphilbert) [inline end2end_test.go]: remove the commented out code

**Review by akeemphilbert — CHANGES_REQUESTED** [review]

**Review by akeemphilbert — APPROVED** [review]

File: end2end_test.go [hunk 3]
```diff
@@ -210,10 +210,10 @@ func aModelShouldBeAddedToTheProjection(arg1 string, details *godog.Table) error
 				}
 				if !strings.EqualFold(column.DatabaseTypeName(), cell.Value) {
 					return fmt.Errorf("expected to get type '%s' got '%s'", cell.Value, column.DatabaseTypeName())
-
 				}
 			//ignore this for now.  gorm does not set to nullable, rather defaulting to the null value of that interface
 			case "Null", "Default":
+
```
Comment (akeemphilbert) [inline end2end_test.go:216]: Null check works for mysql and postgres just not sqlite. Make an exception only for sqlite

**Review by akeemphilbert — CHANGES_REQUESTED** [review]

File: projections/projections_test.go [hunk 4]
```diff
@@ -1207,3 +1207,114 @@ components:
...
+		gormDB := app.DB()
+		if !gormDB.Migrator().HasTable("Blog") {
+			t.Fatal("expected to get a table 'Blog'")
+		}
+
+		columns, _ := gormDB.Migrator().ColumnTypes("Blog")
+
+		for _, c := range columns {
+			switch *driver {
+			case "mysql":
```
Comment (akeemphilbert) [inline projections/projections_test.go]: check for not sqlite

**Review by akeemphilbert — CHANGES_REQUESTED** [review]

**Review by akeemphilbert — APPROVED** [review]

**Pull request merged**